provision/start-agent.sh

```diff
@@ -7,12 +7,19 @@ vault login -method=github
 AS_AGENT="sudo -u buildkite-agent -H"
 
 ## Read docker info from vault and login
-DOCKER_USERNAME=`vault read -field=username secret/vimc-robot/dockerhub`
+DOCKER_USERNAME=$(vault read -field=username secret/vimc-robot/dockerhub)
 vault read -field=password secret/vimc-robot/dockerhub | \
   $AS_AGENT docker login -u $DOCKER_USERNAME --password-stdin
 
+## Once we retire the registry this section goes away
+VIMC_DOCKER_URL=docker.montagu.dide.ic.ac.uk:5000
+VIMC_DOCKER_USERNAME=vimc
+VIMC_DOCKER_PASSWORD=$(vault read -field=password /secret/vimc/registry/vimc)
+echo $VIMC_DOCKER_PASSWORD | \
+    $AS_AGENT docker login -u $VIMC_DOCKER_USERNAME --password-stdin $VIMC_DOCKER_URL
```

Comment:
If we're logging into docker and at a particular registry what does that mean when we call `docker push` from builds? Do the push have to specify if we want to push/pull with this registry?

Reply:
Actually ignore that - I've seen how you're doing it in the orderly PR.